docs/quick-start/transmitters/lua-howto.md

```diff
@@ -136,6 +157,10 @@ Pressing the `[Bind]` button activates binding mode for traditional binding. Thi
 
 Pressing the `[BLE Joystick]` selection activates BluetoothLE Joystick mode which allows connection to simulators through the bluetooth of your computer. Reboot or change models to exit this mode.
 
+### Other Devices
+
+This folder will allow you to switch between the TX module settings (above) or Receiver-specific settings, like Model Match ID or the Telemetry Transmit Power for PA/LNA-equipped Receivers. You can also check the version flashed into the currently connected Receiver through this folder. You can also use the Model Loaning and Return Model features from this folder.
```

Comment:
The Other Devices folder, if present, allows changing the configuration of other connected devices, such as the current receiver. Options can include Telemetry Power, Diversity Mode, and the Loan / Return Model features.